src/main/java/no/nav/tag/dittNavArbeidsgiver/services/altinn/AltinnService.java

```diff
@@ -73,20 +75,27 @@ public AltinnService(AltinnConfig altinnConfig, RestTemplate restTemplate, Token
     @Cacheable(ALTINN_CACHE)
     public List<Organisasjon> hentOrganisasjoner(String fnr) {
         String filterParamVerdi = "Type+ne+'Person'+and+Status+eq+'Active'";
-
         if (unleash.isEnabled("arbeidsgiver.ditt-nav-arbeidsgiver-api.bruk-altinn-proxy")) {
             Map<String, String> parametre = new ConcurrentHashMap<>();
             parametre.put("$filter", filterParamVerdi);
             log.info("Henter organisasjoner fra Altinn via proxy");
-
+        try {
```

Comment:
lurer på om dette også er dobbel unntakshåndtering, siden det er try catch er i getReporteesviaProxy og? eller er dette for å sende det videre til frontenden?

Reply:
fjernet denne også